data/css/modules/card/_default.scss

```diff
@@ -209,6 +209,19 @@
                 padding: 20px;
             }
         }
+
+        // Video/audio icon sizing
```

Comment:
Here are the specs
https://endlessm.invisionapp.com/d/main#/console/8756792/238130473/preview

The only different size is in the smaller default postcard (Card Height B)